index.html

```diff
@@ -1608,7 +1608,35 @@ <h3>Defining 'string'</h3>
 	</div>
 </section>
 
+<section id="char_whitespace" class="subtopic">
+	<h3>Whitespace characters</h3>
+	
+	<div class="xref"><span class="seealso">See also</span>
+	<p>[[[#markup_identifiers]]].</p>
+	</div>
 
+	<p>
+		Whitespace characters are characters that represent horizontal or vertical space in typography. Whitespace characters can have different visual effects: some whitespace characters have no visible effect, while others represent larger, smaller, or variable amounts of space on the page.
+	</p>
+	
+		<div class="req" id="char_define_whitespace">
+			<p>
+				Some specifications that define formal languages (examples include <a href="https://html.spec.whatwg.org/multipage/syntax.html#syntax">HTML</a> or <a href="https://www.w3.org/TR/css-syntax/#whitespace">CSS</a>) will choose to specify ASCII whitespace as part of their grammar. Specifications that deal more generally with text will choose to follow Unicode's definition instead.
```

Comment:
Hmm... reading my suggestion again, I think we need to define ASCII whitespace. This also doesn't provide quite enough guidance. I don't have time right now to make suggestions but will try to come back to this in a bit. @r12a what do you think?